og.module

```diff
@@ -44,6 +45,62 @@ function og_entity_insert(EntityInterface $entity) {
 }
 
 /**
+ * Implements hook_og_group_created().
+ */
+function og_og_group_created($entity_type, $bundle) {
```

Comment:
seem to me much more easy to pass this logic into a hook rather keep the code in the group manager. I'll try something.